CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -524,43 +662,50 @@ end
 local function tweenBackgroundColor(frame, endColor, duration)
 	local t = 0
 	local prevTime = tick()
-	local startColor = frame.BackgroundColor3
+	local colorProp = frame:IsA("ImageLabel") and "ImageColor3" or "BackgroundColor3"
```

Comment:
It seems a bit misleading that the function called `tweenBackgroundColor` could secretly be tweeing the ImageColor instead